common/schema-model/src/main/java/io/smallrye/graphql/schema/model/Schema.java

```diff
@@ -33,7 +34,13 @@
 
     private Map<String, ErrorInfo> errors = new HashMap<>();
 
-    public Schema() {
+
+    /**
+     * Constructor
+     * @param isFederated indicates if the schema is a federated schema
+     */
+    public Schema(boolean isFederated) {
+        this.isFederated = isFederated;
```

Comment:
All the other schema details are set with setters, not with a constructor parameter. Just for uniformity, we should stick to that, or is there a good reason to change it?

Reply:
Yes for Quarkus we need a public constructor and getters and setters